webapp/src/core.ts

```diff
@@ -456,14 +456,15 @@ export function scrollIntoView(item: JQuery, margin = 0) {
     }
 }
 
-export function initializeFocusTabIndex(element: Element) {
-    if (element !== document.activeElement && element.contains(document.activeElement)) {
+export function initializeFocusTabIndex(element: Element, allowResetFocus = false) {
+    if (!allowResetFocus && element !== document.activeElement && element.contains(document.activeElement)) {
```

Comment:
It is part of the "Fixed a bug where the focus was not given to the correct element in a modal in case of the UI was empty at a moment because of a "loading in progress" behavior." commit [https://github.com/Microsoft/pxt/pull/2426/commits/e3c88971f81dc62072cc245a47656eb495f33843](https://github.com/Microsoft/pxt/pull/2426/commits/e3c88971f81dc62072cc245a47656eb495f33843).

For the Select Language modal, when we open it for the first time, a "Loading..." splash screen appear (visible only with a breakpoint or with a very slow internet connection), and only then the languages list is displayed. It means that the modal content changed at a moment.

But in this method, we made a condition that avoid to change reset the focus to the first interactive element. It is useful for the Share modal, but it generate this problem for the Select Language.
The idea of the allowResetFocus is to allow to move the focus to another element if a modal's content change. This argument is true for Select Language, but false for Share modal for example.